terraform/variables.tf

```diff
@@ -460,3 +460,66 @@ variable "observer_node_counts" {
     sa-east-1      = 0
   }
 }
+
+variable "backup_lambda_binary" {
+  default = "BackupLamba"
+  description = "The BackupLambda binary"
+}
+
+variable "backup_lambda_binary_path" {
+    default = "BackupLambda"
+    description = "Full path to the binary for the BackupLambda"
+}
+
+variable "backup_lambda_binary_url" {
+  description = <<DESCRIPTION
+URL to retrieve the Backup Lambda binary from.
+DESCRIPTION
+  default    = "https://github.com/Eximchain/BackupLambda/releases/download/0.1/BackupLambda"
+}
+
+variable "backup_lambda_ssh_private_key" {
+  description = <<DESCRIPTION
+SSH private key to be used for authentication.
+DESCRIPTION
+  default     = ""
+}
+
+variable "backup_lambda_ssh_private_key_path" {
+  description = <<DESCRIPTION
+Path to the SSH private key to be used for authentication.
+Ensure this keypair is added to your local SSH agent so provisioners can
+connect.
+Example: ~/.ssh/terraform
+DESCRIPTION
+  default     = ""
+}
+
+# this is the lambda zip, must be a relative path
+# eg "BackupLambda.zip"
+variable "backup_lambda_output_path" {
+    default = "BackupLambda.zip"
+    description = "Relative path to the BackupLambda zip, which will be generated from the backup binary"
+}
+
+# output prefix of encrypted SSH key, region will be appended to the filename
+variable "enc_ssh_path" {
+    default = "enc_ssh"
+    description = "Full path to the encrypted SSH key to be generated, region will be appended to the filename"
+}
+
+# key on S3 bucket
+variable "enc_ssh_key" {
+    default = "enc_ssh"
+    description = "The key to access the encrypted SSH key on the S3 bucket"
+}
+
+variable "backup_interval" {
+    default     = ""
+    description = <<DESCRIPTION
+Schedule expression for backup event, see https://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html for examples
```

Comment:
Thanks for adding this link. I was going to ask you to do that but you beat me to it.